prbt_hardware_support/test/unittests/unittest_modbus_adapter_brake_test.cpp

```diff
@@ -290,27 +290,40 @@ TEST_F(ModbusAdapterBrakeTestTest, testModbusWithoutApiVersion)
  * }
  *
  * Test Sequence:
- *  1. Set required state to false
- *  2. Publish modbus message without a brake test status.
+ *  1. Publish modbus message without a brake test status.
  *
  * Expected Results:
- *  1. The service returns false
- *  2. The state has not changes (i. e. is still false)
```

Comment:
Would the old test still work? I mean:
```
1. Set required state to false
￼2. Publish modbus message without a brake test status.
[...]
2. The state has not changes (i. e. is still false)
```
I think this would be a better test here. It can not always be unknown

Reply:
We know from the other tests that the change works